examples/core/Graphormer/main.py

```diff
@@ -38,14 +38,18 @@
     get_polynomial_decay_schedule_with_warmup,
 )
 
+# Instantiate an accelerator object to support distributed
+# training and inference.
+accelerator = Accelerator()
+
 
 def train_epoch(model, optimizer, data_loader, lr_scheduler):
     model.train()
     epoch_loss = 0
     list_scores = []
     list_labels = []
     loss_fn = nn.BCEWithLogitsLoss()
-    for iter, (
+    for counter, (
```

Comment:
After L96 was changed to `epoch_loss /= len(data_loader)`, this counter is not used, consider remove?